### ISSUE TITLE
`@automattic/popup-monitor` is missing a dependency on `@babel/runtime`

### ISSUE THREAD

**Issue by anomiex** [Automattic/wp-calypso#64034]
The package `@automattic/popup-monitor` attempts to use `@babel/runtime` from `dist/cjs/index.js`, but does not declare a dependency or peer dependency on that package.

## Reproduction

With npm:

1. Create a temporary directory, and cd into it.
2. `npm add @automattic/popup-monitor`
3. `node -e 'const x = require( "@automattic/popup-monitor" ); console.log( typeof x );'`

With yarn:

1. Create a temporary directory, and cd into it.
2. `echo '{}' > package.json`
3. `yarn set version stable`
4. `yarn add @automattic/popup-monitor`
5. `yarn node -e 'const x = require( "@automattic/popup-monitor" ); console.log( typeof x );'`

With pnpm:

1. Create a temporary directory, and cd into it.
2. `echo 'hoist-pattern=[]' > .npmrc`
3. `pnpm add @automattic/popup-monitor`
4. `node -e 'const x = require( "@automattic/popup-monitor" ); console.log( typeof x );'`

## Expected behavior

Output along the lines of
```
object
```

## Actual behavior

With npm or pnpm:
```
Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'
Require stack:
- /tmp/test/node_modules/.pnpm/@automattic+popup-monitor@1.0.1/node_modules/@automattic/popup-monitor/dist/cjs/index.js
```

With yarn:
```
Error: @automattic/popup-monitor tried to access @babel/runtime, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

Required package: @babel/runtime (via "@babel/runtime/helpers/interopRequireDefault")
Required by: @automattic/popup-monitor@npm:1.0.1 (via /tmp/test/.yarn/cache/@automattic-popup-monitor-npm-1.0.1-9880e83515-3e0b4a03d9.zip/node_modules/@automattic/popup-monitor/dist/cjs/)
```

**Comment by simison** [comment]
cc @Automattic/team-calypso

**Comment by jsnajdr** [comment]
Hi @anomiex do you want to do a PR? It should be a trivial one.

**Comment by anomiex** [comment]
If no one beats me to it I will once I get some other stuff off my plate. But I don't want to [lick the cookie](https://devblogs.microsoft.com/oldnewthing/20091201-00/?p=15843) if someone else wants to take care of it.

**Comment by daledupreez** [comment]
Closing this issue as it was fixed by #64243.
